### PR TITLE
Enrichment excluded IOCs (Allow-list) layouts update

### PR DESCRIPTION
**Pull request by Ni-Knight**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11323

## Description
Updated layouts for enrichment excluded indicators.

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/CommonTypes/ReleaseNotes/3_5_12.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+- Updated the layout to support enrichment excluded indicators.
+
+##### URL Indicator
+- Updated the layout to support enrichment excluded indicators.
+
+##### Email Indicator
+- Updated the layout to support enrichment excluded indicators.
+
+##### IP Indicator
+- Updated the layout to support enrichment excluded indicators.
```
Comment (ShirleyDenkberg) [inline Packs/CommonTypes/ReleaseNotes/3_5_12.md:17]:
```suggestion
Updated the layout to support enrichment excluded indicators.

##### Domain Indicator
Updated the layout to support enrichment excluded indicators.

##### URL Indicator
Updated the layout to support enrichment excluded indicators.

##### Email Indicator
Updated the layout to support enrichment excluded indicators.

##### IP Indicator
Updated the layout to support enrichment excluded indicators.
```

**Comment by ShirleyDenkberg** [comment]
@altmannyarden @idovandijk Doc review completed.

File: Packs/CommonScripts/Scripts/EnrichExcludeButton/EnrichExcludeButton.yml [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+enabled: true
+scripttarget: 0
+subtype: python3
+runonce: false
+dockerimage: demisto/python3:3.11.9.106403
+runas: DBotWeakRole
+engineinfo: {}
+tests:
+- No tests (auto formatted)
+fromversion: 6.10.0
```
Comment (altmannyarden) [inline Packs/CommonScripts/Scripts/EnrichExcludeButton/EnrichExcludeButton.yml:18]: Please change the fromversion

File: Packs/CommonTypes/ReleaseNotes/3_5_12.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
+
+#### Layouts
+
+##### File Indicator
+- Updated the layout to support enrichment excluded indicators.
```
Comment (altmannyarden) [inline Packs/CommonTypes/ReleaseNotes/3_5_12.md:5]: I think we should change the note for something more meaningful, for example - Added a new Boolean field to set enrichment excluded indicators.

**Review by altmannyarden — APPROVED** [review]

**Pull request merged**